test/openshift/e2e/parallel/1-055_validate_notification_controller/03-errors.yaml

```diff
@@ -27,10 +34,3 @@ kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: example-argocd-argocd-notifications-controller
----
```

Comment:
@ciiay Any reason behind placing this assertion on the top?

Reply:
Will remove it.